src/main/java/DeadlineTask.java

```diff
@@ -0,0 +1,47 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class DeadlineTask extends Task {
+    private final String taskTitle;
+    private final String taskType = "D";
+    private boolean markedStatus;
+    private LocalDateTime deadline;
+
+    public DeadlineTask (String taskTitle, LocalDateTime deadline) {
```

Comment:
Not a naming violation but I noticed an extra space before (String taskTitle, ...) here 😄.
```suggestion
    public DeadlineTask(String taskTitle, LocalDateTime deadline) {
```